docs(GKO): update Kubernetes operator context doc (#977)
gravitee-io/gravitee-docs: pages/apim/3.x/kubernetes/apim-kubernetes-operator-installation-cluster.adoc

This section provides steps on how to deploy the Gravitee Kubernetes Operator (GKO) on an existing APIM-ready Kubernetes cluster.

NOTE: If your architecture requires the management of multiple Kubernetes clusters with a different set of APIs for each cluster, you should deploy the GKO separately on each cluster. Follow the deployment process described below for each cluster deployment.

== Prerequisites

=== Prerequisites

Before you start, you need to have the following software set up and running on your machine:

. https://kubernetes.io/docs/tasks/tools/#kubectl[Kubectl].
. https://helm.sh/docs/intro/install/[Helm v3].

> Moreover, ensure that you have access to an APIM-ready Kubernetes cluster.

Reviewer: ```suggestion
You also need access to an APIM-ready Kubernetes cluster.
```

Run the following command in your command-line tool (the working directory does not matter) to deploy the GKO on the cluster of your current Kubernetes context:

....
helm repo add graviteeio https://helm.gravitee.io

# If you already have the repository, update it using the following command.
#  helm repo update graviteeio

helm upgrade --install --create-namespace gko graviteeio/gko -n gko-system

# By default the operator is configured to manage all the custom resources in the cluster.
# To manage only the custom resources in the namespace where the operator is deployed, set the following property to false using the following command.
# helm upgrade --install --create-namespace gko graviteeio/gko -n gko-system --set manager.scope.cluster=false
....

The operation is quick and the successful command-line output should be similar to one in the example below:

....

> Thank you for installing The Gravitee Kubernetes Operator Helm Chart !

Reviewer: ```suggestion
Thank you for installing The Gravitee Kubernetes Operator Helm Chart!
```
